zebra/zebra_rib.c

```diff
@@ -3154,6 +3157,8 @@ void rib_sweep_table(struct route_table *table)
 			if (!CHECK_FLAG(re->flags, ZEBRA_FLAG_SELFROUTE))
 				continue;
 
+			if (now - re_time < 0)
```

Comment:
Could we have a comment about this bit - to say "don't remove things that we've learned about since the startup marker/point" ?

This is unconditional, right - so any caller through this api will have this new behavior of testing the timestamp? Is that fine - to not make this conditional on the use of the new startup argument?

Reply:
updated to make clearererer